content/en/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -57,49 +57,58 @@ A Pod can specify a set of shared storage {{< glossary_tooltip text="Volumes" te
 {{< /note >}}
 
-Pods do not, by themselves, self-heal. If a Pod is scheduled to a Node that fails, or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a higher-level abstraction, called a *Controller*, that handles the work of managing the relatively disposable Pod instances. Thus, while it is possible to use Pod directly, it's far more common in Kubernetes to manage your pods using a Controller. See [Pods and Controllers](#pods-and-controllers) for more information on how Kubernetes uses Controllers to implement Pod scaling and healing.
+Pods do not, by themselves, self-heal. If a Pod is scheduled to a Node that fails, or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a higher-level abstraction, called a controller, that handles the work of managing the relatively disposable Pod instances. Thus, while it is possible to use Pod directly, it's far more common in Kubernetes to manage your pods using a controller. See [Pods and controllers](#pods-and-controllers) for more information on how Kubernetes uses controllers to implement workload scaling and healing.
 
-### Pods and Controllers
+### Pods and controllers
 
-A Controller can create and manage multiple Pods for you, handling replication and rollout and providing self-healing capabilities at cluster scope. For example, if a Node fails, the Controller might automatically replace the Pod by scheduling an identical replacement on a different Node.
+You can use workload resources to create and manage multiple Pods for you. Inside the cluster's control plane, a controller for that resource handles replication and rollout and automatic healing in case of Pod failure. For example, if a Node fails, a controller notices that Pods on that Node have stopped working and creates a replacement Pod. The scheduler places the replacement Pod onto a healthy Node.
 
-Some examples of Controllers that contain one or more pods include:
+Here are some examples of workload resources that manage one or more Pods:
 
-* [Deployment](/docs/concepts/workloads/controllers/deployment/)
-* [StatefulSet](/docs/concepts/workloads/controllers/statefulset/)
-* [DaemonSet](/docs/concepts/workloads/controllers/daemonset/)
+* {{< glossary_tooltip text="Deployment" term_id="deployment" >}}
+* {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
+* {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}
 
-In general, Controllers use a Pod Template that you provide to create the Pods for which it is responsible.
+Workload resources use a _pod template_, that you provide, to create the Pods for which it is responsible.
```

Comment:
The original text seems more explicit, but with some tweaks?
Style nits:
- Capitalize controllers or not
- PodTemplate is a thing, https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#podtemplate-v1-core
Is this referring to the resource or `Pod template`?

Original text:
`In general, controllers use a Pod Template that you provide to create the Pods for which it is responsible.`

Possible edit:
Controllers create and manage Pods applying a user defined `PodTemplate`.